.chain-interactions/send-transactions/interoperability/transfer-assets-into-polkadot.md

```diff
@@ -0,0 +1,221 @@
+
+```ts title="index.ts"
+--8<-- 'code/chain-interactions/send-transactions/interoperability/transfer-assets-into-polkadot/index.ts:85:89'
+```
+
+## Supported Assets and Destinations
+
+Snowbridge can bridge any ERC-20 token from Ethereum, but the destination chain must have that asset registered as a foreign asset to receive it.
+
+!!!warning "Verify Foreign Assets"
+    Foreign asset creation on Polkadot Hub is permissionless—anyone can register a foreign asset. Before bridging, verify that the foreign asset registered on Polkadot Hub corresponds to the legitimate token on Ethereum by checking the asset's multilocation and confirming the Ethereum contract address matches the official token contract.
+
+You can query which assets are supported for a specific route using the ParaSpell SDK:
+
+```ts title="index.ts"
+import { getSupportedAssets } from '@paraspell/sdk-pjs';
+
+// Get assets supported for Ethereum → Polkadot Hub transfers
+const supportedAssets = getSupportedAssets('Ethereum', 'AssetHubPolkadot');
+console.log('Supported assets:', supportedAssets.map(a => a.symbol));
+```
+
+--8<-- 'code/chain-interactions/send-transactions/interoperability/transfer-assets-into-polkadot/supported-assets-output.html'
+
+Common supported tokens on Polkadot Hub include:
+
+| Token | Symbol | Description |
+|-------|--------|-------------|
+| Wrapped Ether | WETH | Ethereum's native token wrapped as ERC-20 |
+| Wrapped Bitcoin | WBTC | Bitcoin wrapped as an ERC-20 token |
+| Shiba Inu | SHIB | Popular ERC-20 meme token |
+
+To transfer to a different destination, change the `.to()` parameter and verify the asset is supported:
+
+```ts title="index.ts"
+// Check if WETH is supported on Hydration
+const hydrationAssets = getSupportedAssets('Ethereum', 'Hydration');
+const wethSupported = hydrationAssets.some(a => a.symbol === 'WETH');
+
+if (wethSupported) {
+  await EvmBuilder(provider)
+    .to('Hydration')
+    .currency({ symbol: 'WETH', amount: amount })
+    .address(recipientAddress)
+    .signer(signer)
+    .build();
+}
+```
+
+--8<-- 'code/chain-interactions/send-transactions/interoperability/transfer-assets-into-polkadot/hydration-check-output.html'
+
+???- code "Full Code"
+
+    ```typescript title="index.ts"
+    --8<-- 'code/chain-interactions/send-transactions/interoperability/transfer-assets-into-polkadot/index.ts'
+    ```
+
+!!!info "Transfer Assets Out of Polkadot"
+
+    To transfer assets out of Polkadot, you can use the ParaSpell XCM SDK to build and execute a bridge transfer from Polkadot to Ethereum. Check out the [Polkadot to Ethereum](https://paraspell.github.io/docs/sdk/xcmPallet.html#polkadot-ethereum-transfer){target=\_blank} section of the ParaSpell XCM SDK documentation.
+
+
+## Where to Go Next
+
+- **Read the docs** - Dive deeper into the [ParaSpell XCM SDK](https://paraspell.github.io/docs/sdk/getting-started.html){target=\_blank} and [Snowbridge](https://docs.snowbridge.network/){target=\_blank} documentation
```

Comment:
```suggestion
- **External Documentation** - Dive deeper into the [ParaSpell XCM SDK](https://paraspell.github.io/docs/sdk/getting-started.html){target=\_blank} and [Snowbridge](https://docs.snowbridge.network/){target=\_blank} documentation resources.
```
I don't hate "Read the Docs" but we should give some indication this is taking them to an external site.